lua/wire/wirenet.lua

```diff
@@ -69,14 +73,14 @@ if SERVER then
 				queue_handler_firstidx, queue_handler_lastidx = idx, idx
 				handler_queue[1] = name
 
-				timer.Create("wirelib_net_flush", 0, 1, queue_handler_flush)
+				timer.Create("wirelib_net_flush", 0, 0, queue_handler_flush)
```

Comment:
Is this the rep or delay? Why change to run every tick now?

Reply:
My bad, I always mix up the two and for some reason I didn't consider that reps would be 0.